### ISSUE TITLE
[Desktop] Epic Game Store - can't log in to my account - black screen instead of login panel

### ISSUE THREAD

**Issue by Martina-Neumayer** [brave/brave-browser#10734]
## Description 
I want to log in to my account on the Epic Game Store but I can't.
Instead of the normal login panel I see only a black screen.

## Steps to Reproduce
1. Go to Epic webpage: http://epicgames.com/store/en/ 
2. Click on "sign in" in the upper right corner of the page
3. On the next screen choose "sign in with the epic games" option

## Actual result:
Black screen instead of the normal login panel where I must type my 2FA code.

## Expected result:
A normal page with login panel where I must type my 2FA code.

## Reproduces how often: 
Easily reproduced
Every time I go to the page.

## Brave version (brave://version info)
Brave	1.12.86 Chromium: 84.0.4147.85 (Oficjalna wersja) unknown (64-bitowa)
Version	e20c13e958edc3707db3d26d513780a7f6331a4b-refs/branch-heads/4147@{#837}
Operating system	Linux (Arch)
JavaScript	V8 8.4.371.19

## Version/Channel Information:

- Can you reproduce this issue with the current release?
I don't use this channel

- Can you reproduce this issue with the beta channel?
Yes

- Can you reproduce this issue with the dev channel?
I don't use this channel

- Can you reproduce this issue with the nightly channel?
I don't use this channel

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields?
No

- Does the issue resolve itself when disabling Brave Rewards?
No

- Is the issue reproducible on the latest version of Chrome? 
Don't know, I don't use Chrome

## Miscellaneous Information:
After downgrade Brave to the 1.11.91 version all goes back to normal and I can log in without any problems.

**Comment by Martina-Neumayer** [comment]
Updated today to v.1.12.89 Chromium: 84.0.4147.89 and nothing changed. Stil can not login on Epic Store.

**Comment by Martina-Neumayer** [comment]
Another update and a workaround.

So..
I updated my Brave today to v.1.12.91 Chromium: 84.0.4147.89 (Arch AUR release) and also found a workaround for this damn nasty "unable to login" issue.

All you need to do is go to Chrome Web Store, find an extension called "iNinja VPN & Proxy" and install it.
Next click on this extension in the browser tab, choose from the location menu "New York,USA" and turn the proxy on.
Then go to the Epic Games Store page as usual and now you can login into your account without any problems.
After successful login you can turn the proxy off and you will stil be able to login again.

**Comment by rebron** [comment]
![Screen Shot 2020-09-02 at 11 36 05 AM](https://user-images.githubusercontent.com/5951041/92022695-82ec3800-ed10-11ea-9f31-2b1f67e66a3c.png)
Hi @Martina-Neumayer. one more update ;-) to 1.14.x, sounds like you're on beta channel which we really appreciate. I'm seeing the screenshot/login above. Could be a linux, arch linux specific issue?

**Comment by rebron** [comment]
cc: @ryanbr

**Comment by Martina-Neumayer** [comment]
@rebron Yes, I am using only the beta channel releases. And nope.. it wasn't Arch problem. It later turned out that the cause of the problem were shields. Despite being turned off from under the shield panel/menu, they were still active. I need to disable them by using (from the terminal) an exec with added custom switch, or by using this trick with the "iNinja" addon.

In the meantime Brave was updated few times (I have now v.1.14.73 Chromium: 85.0.4183.83) and is working fine.
No more problems with the Epic Store.

I think we can close this issue.
